modules/programs/rbrowser/default.nix

```
@@ -1,20 +1,60 @@
-{ mode, config, pkgs, lib, ... }:
+{ mode, config, home-manager, lib, pkgs, ... }:
 
 with lib;
 let
   cfg = config.soxin.programs.rbrowser;
 
-  browserSubmodule = { name, ... }: {
-    options = {
-      enable = mkOption {
-        default = true;
-        type = types.bool;
-        description = ''
-          Whether to enable the ${name} integration with rbrowser.
-        '';
+  browserSubmodule =
+    let
+      getFromDagName = dagName: index:
+        let
+          hasAt = lib.stringAsChars (x: if x == "@" then x else "") dagName == "@";
+          name_profile = builtins.head (builtins.tail (builtins.split "^(.*)@(.*)$" dagName));
```

```suggestion
          nameProfile = builtins.head (builtins.tail (builtins.split "^(.*)@(.*)$" dagName));
```
Maybe let's keep the same naming conventions everywhere?